lib/Common/Memory/Recycler.h

```diff
@@ -1502,7 +1502,7 @@ class Recycler
     template <typename SmallHeapBlockAllocatorType>
     void RemoveSmallAllocator(SmallHeapBlockAllocatorType * allocator, size_t sizeCat);
     template <ObjectInfoBits attributes, typename SmallHeapBlockAllocatorType>
-    char * SmallAllocatorAlloc(SmallHeapBlockAllocatorType * allocator, size_t sizeCat);
+    char * SmallAllocatorAlloc(SmallHeapBlockAllocatorType * allocator, size_t sizeCat, size_t size);
```

Comment:
will do, this needs pogo build, most of these function should be inlined so no stack arguments passing overhead I think

---
In reply to: [60997858](https://github.com/Microsoft/ChakraCore/pull/711#discussion_r60997858) [](ancestors = 60997858)